src/HtmlGenerator/HtmlGenerator.csproj

```diff
@@ -1265,6 +1265,9 @@
     </None>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="AffiliateLinks.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>

```

Comment:
Please change to PreserveNewest. Never use Always because it breaks build incrementality :)